src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -29,6 +29,8 @@ interface Props {
   voiceFocusOnLabel?: string;
   /** The label that will be shown when the current input audio is not an Amazon Voice Focus device, it defaults to `Enable Amazon Voice Focus`. */
   voiceFocusOffLabel?: string;
+    /** A boolean that determines whether or not to include additional audio devices to the control bar */
```

Comment:
Should we add some information on what additional devices are and change the prop name since now this will be public? 

It will help builders else not sure how they can understand what additional devices are. I could not think of a good name which will explain directly but we can have a better name to reflect the devices use it seems.